chat-plugins/trivia.js

```diff
@@ -418,27 +406,33 @@ var commands = {
 	// trivia game commands
 	new: function (target, room) {
 		if (room.id !== 'trivia' || !this.can('broadcast', null, room) || !target) return false;
-		Trivia.getInstance().startSignups(target, room, this);
+		if (!room.Trivia) room.Trivia = Trivia;
```

Comment:
Though that'd still require a check in each command to check if ``this.Trivia`` exists or not.